kibana/Dockerfile

```diff
@@ -75,4 +80,16 @@ RUN ./welcome_wazuh.sh
 
 RUN /usr/local/bin/kibana-docker --optimize
 
+
+# Set Wazuh custom login image
```

Comment:
Great welcome screen, but we are not going to customize it yet.

Reply:
Moved